.github/workflows/auto-remove-label.yml

```diff
@@ -0,0 +1,11 @@
+name: 🤖 Auto remove label on closed issue
+on:
+  issues:
+    types: [closed]
+
+jobs:
+  remove_label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.6.0
+      - uses: taiga-family/ci/actions/auto-remove-label@1.12.0
```

Comment:
"state: need triage" label is not used inside Maskito.

I think that we can drop this workflow.

Reply:
Now it's used
https://github.com/taiga-family/maskito/labels